examples/ec2/securitygrouprule-self-true.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: ec2.aws.upbound.io/v1beta1
```

Comment:
We had better add the `meta.upbound.io/example-id: ec2/v1beta1/securitygrouprule` annotations to the resource manifests.